CHANGELOG.unreleased.md

```diff
@@ -36,6 +36,7 @@
 - @vates/nbd-client patch
 - @xen-orchestra/backups patch
 - @xen-orchestra/vmware-explorer patch
+- xapi-explore-sr major
```

Comment:
Now that I think about it, we don't need to do another release for it, nothing has changed for the users.

The only important thing is that the code is now up-to-date regarding `xen-api` and if it can be released when necessary without issue.

You can remove this line.